src/util-inl.h

```diff
@@ -377,14 +377,12 @@ v8::MaybeLocal<v8::Value> ToV8Value(v8::Local<v8::Context> context,
   if (isolate == nullptr) isolate = context->GetIsolate();
   v8::EscapableHandleScope handle_scope(isolate);
 
-  MaybeStackBuffer<v8::Local<v8::Value>, 128> arr(vec.size());
-  arr.SetLength(vec.size());
+  v8::LocalVector<v8::Value> arr(isolate, vec.size());
```

Comment:
When I attempted to change these before the feedback was that this was likely to be a performance hit. The suggetion at the time was to have an alternative implmementation of `MaybeStackBuffer<v8::Local<T>>` that uses `v8::LocalVector<T>` internally when the size exceeds the stack limit.